flux-operator.yaml

```
@@ -28,7 +83,19 @@ subpackages:
           ln -sf /usr/bin/flux-operator ${{targets.contextdir}}/flux-operator
     test:
       pipeline:
-        - runs: stat /flux-operator
+        - runs: |
+            stat /flux-operator
+
+  - name: ${{package.name}}-mcp-compat
+    description: compat package for flux-operator-mcp
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.contextdir}}"
+          ln -sf /usr/bin/flux-operator-mcp ${{targets.contextdir}}/flux-operator-mcp
+    test:
+      pipeline:
+        - runs: |
+            stat /flux-operator-mcp
```

Not a blocker, small enhancement for symlink validation:
```suggestion
            test "$(readlink /flux-operator-mcp)" = "/usr/bin/flux-operator-mcp"
```